conmon/conmon.c

```diff
@@ -429,26 +496,8 @@ int main(int argc, char *argv[])
 
 					ninfo("read a chunk: (fd=%d) '%s'", mfd, buf);
 
-
-					/* Prepend the CRI-mandated timestamp and other metadata. */
-					/*
-					 * FIXME: Currently this code makes the assumption that
-					 *        @buf doesn't contain any newlines (since the CRI
-					 *        requires each line to contain a timestamp). This
-					 *        is an /okay/ assumption in most cases because
-					 *        ptys generally have newline-buffered output.
-					 */
-					int rc = set_k8s_timestamp(tsbuf, TSBUFLEN, "stdout");
-					if (rc < 0) {
-						nwarn("failed to set timestamp");
-					} else {
-						if (write(logfd, tsbuf, TSBUFLEN) != TSBUFLEN) {
-							nwarn("partial/failed write ts (logFd)");
-						}
-					}
-					/* Log all output to logfd. */
-					if (write(logfd, buf, num_read) != num_read) {
-						nwarn("partial/failed write (logFd)");
+					if (write_k8s_log(logfd, "stdout", buf, num_read) < 0) {
```

Comment:
@cyphar Adding the '\0' is only for making the debug logs better :) I think we will fine even if we just drop the ninfo like you said. The actual log files looked good in my manual testing with a few different types of pods. Unfortunately, the e2e tests ran into unrelated issues on my machine that I am still debugging. So I will ask @runcom to run the suite on his machine.